GameWindow.java

```diff
@@ -279,65 +286,73 @@ public void init() {
-
-		// WAVEDASH
-		// at frame four of the jump is the airdodge, aka 4/60 of a second, aka 1/15 of
-		// a second
-		// the note is moved BY the tick method, which y=y+1 every 1/60th of a second
-		// therefore, y of the downleft and r is 4 y places away from the y?
-		// this doesn't necessarily work because the hitboxes are 32 x 32, half of it is
-		// 16, add 4?20?
-		RnoteFW = new Note("r", 348, -74, this);
-		YnoteFW = new Note("y", 339, -54, this);
-		stickDLeftnoteFW = new Note("stickDLeft", 264, -76, this);
-
-		// MULTISHINE
-		BnoteMS = new Note("b", 333, -37, this);
-		XnoteMS = new Note("x", 359, -51, this);
-		stickDownnoteMS = new Note("stickDown", 264, -40, this);
-		stickDownnoteMS2 = new Note("stickDown", 264, -55, this);
-		stickDownnoteMS3 = new Note("stickDown", 264, -70, this);
-
-		// NAIR
-		AnoteN = new Note("a", 347, -40, this);
-		stickDownnoteN = new Note("stickDown", 264, -30, this);
-		RnoteN = new Note("r", 348, -71, this);
-		YnoteN = new Note("y", 339, -10, this);
-
-		// Ledgedash
-		RnoteLD = new Note("r", 348, -71, this);
-		YnoteLD = new Note("y", 339, -54, this);
-		stickDRightnoteLD = new Note("stickDRight", 264, -43, this);
-		stickleftnoteLD = new Note("stickLeft", 264, -43, this);
-
-		// Short hop double laser
-
-	/*	Anote = new Note("a", 347, -40, this);
-		Bnote = new Note("b", 333, -35, this);
-		Xnote = new Note("x", 359, -51, this);
-		Ynote = new Note("y", 339, -54, this);
-		Lnote = new Note("l", 258, -71, this);
-		Rnote = new Note("r", 348, -71, this);
-		Znote = new Note("z", 367, -62, this);
-		cUpnote = new Note("cUp", 327, -10, this);
-		stickUpnote = new Note("stickUp", 264, -43, this);
-		stickDLeftnote = new Note("stickDLeft", 264, -43, this);
-		stickDownnote = new Note("stickDown", 264, -43, this);
-*/
+		pauseIcon = p.grabImage(1,1,32,32);
+		title = t.grabImage(1, 1, 32,32);
+		
+		
+		
+		//INITIALIZES notes for each tech, varies in position depending on the technique
+		
+		
+		
+		//WAVEDASH
+		//at frame four of the jump is the airdodge, aka 4/60 of a second, aka 1/15 of a second
+		//the note is moved BY the tick method, which y=y+1 every 1/60th of a second
+		//therefore, y of the downleft and r is 4 y places away from the y?
+		//this doesn't necessarily work because the hitboxes are 32 x 32, half of it is 16, add 4?20?
+		RnoteFW = new Note("r", 328, -74, this);
+		YnoteFW = new Note("y", 319, -54, this);
+		stickDLeftnoteFW = new Note("stickDLeft", 244, -74, this);
+		
+		
+		
+		//MULTISHINE
+		//here is where the notes are specified in positions
+		
+		
+		
+		//NAIR
+		AnoteN = new Note("a", 327, -40, this);
+		stickDownnoteN = new Note("stickDown",244,-30,this);
+		RnoteN = new Note("r", 328, -71, this);
+		YnoteN = new Note("y", 319, -10, this);
+		
+		//Ledgedash
+		RnoteLD = new Note("r", 328, -71, this);
+		YnoteLD = new Note("y", 319, -54, this);
+		stickDRightnoteLD = new Note("stickDRight", 244, -43, this);
+		stickleftnoteLD = new Note("stickLeft",244,-43,this);

```

Comment:
If my understanding is correct, each technique is made up of the listed notes in order, right?

If so, this is a perfect candidate for an array. We could do something like this:

```java
// In your class variable definitions
private Note[] technique;

// When selecting a technique
if (technique == "Nair") {
  technique = { new Note("a", 327, -40, this), new Note("stickDown", 244, 30, this), ...and so forth }
} else if (technique == "Ledgedash") {
  //etc
}
```

Ideally you'd factor this logic elsewhere (e.g. into a Technique class) but that isn't a huge priority in comparison to at least getting them into an array.